src/utils/workspace.ts

```diff
@@ -154,4 +154,5 @@ export async function ensureFolderIsOpen(fsPath: string, actionContext: IActionC
             throw new Error(message);
         }
     }
+    return fsPath;
```

Comment:
fsPath is the same thing passed in. You have to actually return `folder.fsPath` or whatever

Reply:
Oh shoot, of course, I got rid of the subDeploy setting earlier >< Thanks for catching that